internal/proxy/directors.go

```diff
@@ -177,6 +179,59 @@ func (h DirectorSetHeaders) Director(request *http.Request) error {
 	return nil
 }
 
+type headers map[string]string
```

Comment:
It will be better have some longer name for type, because headers - very good name for local variable. And shadow type name by value name is bad practice.